src/Pages/Auth/Login/index.js

```diff
@@ -21,9 +24,36 @@ function Login(props) {
     history.push("/login")
   }
 
+  function handleSubmit(e) {
+    e.preventDefault()
+    console.log(formData)
+  }
+
+  const [formData, setFormData] = useState({
+    email: "",
+    password: ""
+  })
+
+  function onChange(e) {
+    setFormData({ ...formData, [e.target.name]: e.target.value })
```

Comment:
Very nice! I would suggest as an alternative for eliminating the dependency to the formData variable
```suggestion
    setFormData((prevFormData) => ({ ...prevFormData, [e.target.name]: e.target.value }))
    

```